doc/changelog/08-cli-tools/14558-coq_makefile_doc.rst

```diff
@@ -0,0 +1,17 @@
+- **Changed:**
+  Change in the syntax of `_CoqProject` files:
+  - `-arg` is now handled by `coq_makefile` and not make
+  - all unquoted `#` start a line comment
+  (`#14558 <https://github.com/coq/coq/pull/14558>`_,
+  by Columbus240).
+- **Removed:**  
+  The following `coq_makefile` options were deprecated and are now removed:
+  `-extra`, `-extra-phony`, `-custom`, `-no-install`, `-install`, `-no-opt`, `-byte`.
+  The support for subdirectories is also removed.
+  (`#14558 <https://github.com/coq/coq/pull/14558>`_,
+  by Columbus240).
+- **Added**:
+  `coq_makefile` now takes the `-docroot` option as alternative to the
+  `INSTALLCOQDOCROOT` variable
+  (`#14558 <https://github.com/coq/coq/pull/14558>`_,
+  by Columbus240).
```

Comment:
Would you condense this into a single Changed entry for the PR?  The changelog is meant to alert users to changes in a general way.  It doesn't need to be so specific--the doc describes what is now accepted.  High-impact breaking changes are worth mentioning explicitly, though.  Maybe something like "Removed deprecated `coq_makefile` options and subdirectory support; added `-docroot` option; unquoted `#` in `_CoqProject` now always begins a comment".

You could make `coq_makefile` and `_CoqProject` into hyperlinks for the convenience of users.

Reply:
No I think making a single entry is not great for the reader. But from the docs about making changelog entries I got the idea that making a single entry was expected. I wasn’t even sure how to add multiple entries correctly (other than trying out). That’s why I asked.
Thanks for helping me. I overthink things sometimes, leading to weird results.

Reply:
This file becomes part of the "Changes" section of the manual, which has 10 screenfuls of change info for 8.14.  This is quite wordy compared to most of the entries there.

Reply:
> Would you condense this into a single Changed entry for the PR?

@jfehrle This was the case before and it was split out in three. See the discussion above. What you are proposing is to revert this change. Since this PR does multiple types of changes, it makes sense to have multiple types of entries.

> This is quite wordy compared to most of the entries there.

If the PR had been split into three with the same content, no one would have complained about a long changelog. Anyway, if the changelog for a given release is considered to be too long, this is up to the RM / core team to tackle this issue, not to the individual contributors. This may indicate that we release too unfrequently for instance.